src/main/java/frc/robot/subsystems/CoralManipulator.java

```diff
@@ -54,7 +62,38 @@ public CoralManipulator() {
         openLoop = false;
         changedPos = false;
         targetAngle = Degrees.of(0);
-    }
+
+        // Creates a SysIdRoutine
+        routine = new SysIdRoutine(
+            new SysIdRoutine.Config(
+              null,        // Use default ramp rate (1 V/s)
+              Volts.of(4), // Reduce dynamic step voltage to 4 to prevent brownout
+              null,        // Use default timeout (10 s)
+                           // Log state with Phoenix SignalLogger class
+              (state) -> SignalLogger.writeString("state", state.toString())
+           ),
+           new SysIdRoutine.Mechanism(
+            (volts) -> 
+            {
+              pivotMotor.setControl(m_voltReq.withOutput(volts.in(Volts))); 
+              rollerMotor.setControl(m_voltReq.withOutput(volts.in(Volts)));
```

Comment:
can u remove this from manipulator and merge main into this branch, 
![image](https://github.com/user-attachments/assets/a016d5d5-4f0a-4b94-ab40-a4788e9b7cdb)